src/main/java/gov/usds/case_issues/db/repositories/TroubleCaseRepository.java

```diff
@@ -38,7 +43,8 @@
 	public Optional<TroubleCase> findByCaseManagementSystemAndReceiptNumber(CaseManagementSystem caseManager, String receiptNumber);
 
 	@Lock(LockModeType.PESSIMISTIC_WRITE) // might need to be more aggressive when postgresql table-level LOCK is available
-	public Collection<TroubleCase> getAllByCaseManagementSystemAndReceiptNumberIn(CaseManagementSystem caseManager, Collection<String> receiptNumbers);
+	public Collection<TroubleCase> getAllByCaseManagementSystemAndReceiptNumberIn(CaseManagementSystem caseManager,
```

Comment:
Probably not worth more dev time now. However, it might be worth making an issue to batch the calls to `getAllByCaseManagementSystemAndReceiptNumberIn` from [`putIssueList`](https://github.com/usds/case-issue-api/blob/master/src/main/java/gov/usds/case_issues/services/CaseListService.java#L163)

Reply:
Yes, handling more than 30,000 cases in the upload is worthwhile, but also there are some plausible memory/performance issues with that routine if we go big, and we should handle all those things at once.

Reply:
Opened #36 to track.